content/docs/1.7.0/advanced-resources/backing-image/backing-image-encryption.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Backing Image Encryption
+weight: 2
+---
+
+Starting with v1.7.0, Longhorn allows you to encrypt and decrypt a backing image by cloning it. The backing image encryption mechanism utilizes the Linux kernel module `dm_crypt` and the command-line utility `cryptsetup`.
+
+## Clone a Backing Image
+You can clone a backing image using YAML code.
+
+Example of a downloaded backing image:
+
+```yaml
+apiVersion: longhorn.io/v1beta2
+kind: BackingImage
+metadata:
+  name: parrot
+  namespace: longhorn-system
+spec:
+  sourceType: download
+  sourceParameters:
+    url: https://longhorn-backing-image.s3-us-west-1.amazonaws.com/parrot.raw
+  checksum: 304f3ed30ca6878e9056ee6f1b02b328239f0d0c2c1272840998212f9734b196371560b3b939037e4f4c2884ce457c2cbc9f0621f4f5d1ca983983c8cdf8cd9a
+```
+
+Example of YAML code used to clone the sample backing image:
+
+```yaml
+apiVersion: longhorn.io/v1beta2
+kind: BackingImage
+metadata:
+  name: parrot-cloned
+  namespace: longhorn-system
+spec:
+  sourceType: clone
+  sourceParameters:
+    backing-image: parrot
+    encryption: ignore
```

Comment:
We probably can improve that `encryption` is automatically mutated with `ignore` if it is an empty string. WDYT?

Reply:
Agree, I will create a hot fix PR.

Reply:
https://github.com/longhorn/longhorn-manager/pull/2956/files